public/css/style.css

```diff
@@ -119,6 +119,7 @@
     background: rgba(0, 0, 0, 0.8);
     color: white;
     border-radius: 2px;
+    z-index: 999999;
```

Comment:
It's just an arbitrarily large number. Though I assume we won't be drawing
1M SVGs! Datasets that large would be pointless viewed on a scatterplot
anyway. That's something where 2D binning would be useful :)
On 11/02/2015 12:37 AM, "Juan Nunez-Iglesias" <notifications@github.com>
wrote:

> In public/css/style.css
> <https://github.com/microscopium/microscopium-ui/pull/24#discussion_r24411857>
> :
>
> > @@ -119,6 +119,7 @@
> >      background: rgba(0, 0, 0, 0.8);
> >      color: white;
> >      border-radius: 2px;
> > +    z-index: 999999;
>
> Is this assuming that we won't have > 1M points?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/microscopium/microscopium-ui/pull/24/files#r24411857>.
>